praw/models/reddit/mixins/__init__.py

```diff
@@ -112,6 +115,37 @@ def remove(self, spam=False):
         data = {'id': self.thing.fullname, 'spam': bool(spam)}
         self.thing._reddit.post(API_PATH['remove'], data=data)
 
+    def send_removal_message(self, type,  # pylint: disable=redefined-builtin
+                             title, message):
+        """Send a removal message for a Comment or Submission.
+
+        Reddit adds human-readable information about the object to the message.
+
+        :param type: One of 'public', 'private', 'private_exposed'.
+            'public' leaves a stickied comment on the post.
+            'private' sends a Modmail message with hidden username.
+            'private_exposed' sends a Modmail message without hidden username.
+        :param title: The short reason given in the message.
+            (Ignored if type is 'public'.)
+        :param message: The body of the message.
+
+        If ``type`` is 'public', the new :class:`~.Comment` is returned.
+        """
+        # The API endpoint used to send removal messages is different
+        # for posts and comments, so the derived classes specify which one.
+        if self.REMOVAL_MESSAGE_API is None:
+            raise NotImplementedError('ThingModerationMixin must be extended.')
+        url = API_PATH[self.REMOVAL_MESSAGE_API]
+
+        # Only the first element of the item_id list is used.
+        data = {'item_id': [self.thing.fullname],
+                'message': message,
+                'title': title,
+                'type': str(type)}
+
+        ret = self.thing._reddit.post(url, data={'json': dumps(data)})
```

Comment:
I'm not a huge fan of the variable name `ret`. Generally speaking, I would prefer `return_value` over `ret`, but in this particular case, it's really more of a `response`. Thoughts?

Reply:
By now the variable could just be removed for `return self.thing._reddit.post(url, data={'json': dumps(data)}) or None`.